app/src/main/kotlin/de/janhopp/luebeckmensawidget/api/model/Meal.kt

```diff
@@ -15,6 +15,7 @@ data class Meal(
     val widgetName = when {
         vegan -> "🌻 "
         vegetarian -> "🌽 "
+        "Fisch" in allergens.map { it.name } -> "🐟 "
```

Comment:
You could maybe use the code "Fi"? In case of the api possibly including translations sometimes. 🤷
(Hmm, but maybe "Fisch" is better, if we plan to include other locations - like Gießen or Potsdam - where maybe the allergen codes are different?)

Reply:
The code is a good idea, had not thought of translations...

I am not going to think about other locations until the API supports them. The codes should probably be normalized by the API anyway?

Reply:
Uuuf, normalizing the codes is not an easy task, if we want to support a bigger range of locations. But something to think about another time ^^

Reply:
Ah, for the translations - I spoke with @Draculente and the api probably won't include a translation. Or possibly only when the source supplies the other languages. Maybe he can explain why in the issue over on his repo ^^